plugins/modules/azure_rm_aduser.py

```diff
@@ -288,17 +316,23 @@ def exec_module(self, **kwargs):
 
             if self.state == 'present':
 
+                extension_attributes = None
+
+                if self.on_premises_extension_attributes:
+                    extension_attributes=OnPremisesExtensionAttributes(
+                        **self.on_premises_extension_attributes
+	                  )
+
                 if ad_user:  # Update, changed
 
                     password = None
 
                     if self.password_profile:
                         password = PasswordProfile(
                             password=self.password_profile,
-                        )
+                        )      
```

Comment:
delete trailing whitespace
```suggestion
                        )
```